### ISSUE TITLE
A readable example for StreamDeserializer?

### ISSUE THREAD

**Issue by jaskij** [serde-rs/json#1070]
Looking at the [current documentation](https://docs.rs/serde_json/1.0.105/serde_json/struct.StreamDeserializer.html) for `serde_json::Deserializer`, I simply can not figure out what the correct format is for a stream. Is it simply a concatenation of JSON objects?

**Comment by vidhanio** [comment]
It seems to be written as "Values need to be a self-delineating value e.g. arrays, objects, or strings, or be followed by whitespace or a self-delineating value." It just needs to be 0+ whitespace-seperated json values.

**Comment by jaskij** [comment]
Yeah, I figured that out, but the documentation was of no help in that.
